app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -61,11 +61,13 @@ function initializeCommentDropdown() {
   }
 
   function removeCopyListener() {
-    if (isIOSDevice) {
+    if (isIOSDevice() === true) {
```

Comment:
`true` is always `true` in JS, you don't need to check it explicitly and the `isIOSDevice` either returns `true` or `false`